net/dns/resolver/neterr_other.go

```diff
@@ -12,4 +12,4 @@ func networkIsUnreachable(err error) bool { return false }
 // packetWasTruncated returns true if err indicates truncation but the RecvFrom
 // that generated err was otherwise successful. It always returns false on this
 // platform.
-func packetWasTruncated(err error) bool   { return false }
+func packetWasTruncated(err error) bool { return false }
```

Comment:
When I run `go fmt ./...` from the root directory this change gets added but I don't need to include it.